src/apps/chifra/pkg/utils/utils.go

```diff
@@ -27,6 +28,13 @@ func IsTestModeServer(r *http.Request) bool {
 	return r.Header.Get("User-Agent") == "testRunner"
 }
 
+// IsServerWriter tries to cast `w` into `http.ResponseWriter`
+// and returns true if the cast was successful
+func IsServerWriter(w io.Writer) bool {
```

Comment:
I've been thinking that this exact function would be called `utils.IsApiMode()` which I've avoided up until now because I had to query the environment. I'd like to remove two things from the `Globals`: `ApiMode` and `fmt == "api"` if at all possible, but let's not worry about this just yet (I can do this with the change to the Global structure). Just make a TODO, I think for now.

Reply:
I am going to try to remove utils.IsApiMode(). My guess is that it will be easy.